template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs

```diff
@@ -32,12 +33,12 @@ namespace PlayFabCommon
             if (rSettings->ProductionEnvironmentURL.StartsWith(TEXT("https://")))
             {
                 return rSettings->ProductionEnvironmentURL
-                    + urlPath + TEXT("?sdk=") + versionString + TEXT("&engine=") + engineVersion;
+                    + urlPath + TEXT("?sdk=") + versionString + TEXT("&engineVer=") + engineVersion + TEXT("&platform=") + platformString;
             }
             else
             {
                 return TEXT("https://") + rSettings->TitleId + rSettings->ProductionEnvironmentURL
-                    + urlPath + TEXT("?sdk=") + versionString + TEXT("&engine=") + engineVersion;
+                    + urlPath + TEXT("?sdk=") + versionString + TEXT("&engineVer=") + engineVersion + TEXT("&platform=") + platformString;
```

Comment:
Is engineVer the correct key here? The Unity PR changed its string from engineVersion to engine.

Reply:
Also the Unity PR didn't include sdk version. Should it?

Reply:
Oh no engineVer isn't the correct key here I should change it to match the unity

Reply:
Also the unity sdk already included sdk version info in the query string prior to my recent pr

Reply:
Hm. I don't see it but I'll follow up not on this unrelated PR.